### ISSUE TITLE
Could not resolve dependency Error

### ISSUE THREAD

**Issue by selected-pixel-jameson** [lukaskupczyk/ngx-super-croppie#11]
Hello, after updating this project to version 15.0.0 and Angular to version 15.1.1 I'm receiving the following dependency issue related to ngx-super-croppie.

```
npm ERR! Found: rxjs@7.4.0
npm ERR! node_modules/rxjs
npm ERR!   rxjs@"7.4.0" from the root project
npm ERR!   peer rxjs@"^6.5.3 || ^7.4.0" from @angular/common@15.1.0
npm ERR!   node_modules/@angular/common
npm ERR!     @angular/common@"^15.1.0" from the root project
npm ERR!     peer @angular/common@"^15.0.0" from ngx-super-croppie@15.0.0
npm ERR!     node_modules/ngx-super-croppie
npm ERR!       ngx-super-croppie@"^15.0.0" from the root project
npm ERR!   1 more (@angular/core)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer rxjs@"^6.6.7" from ngx-super-croppie@15.0.0
npm ERR! node_modules/ngx-super-croppie
npm ERR!   ngx-super-croppie@"^15.0.0" from the root project
```

**Comment by selected-pixel-jameson** [comment]
It looks like you are still requiring "rxjs": "^6.6.7" in the package.json file.

https://github.com/lukaskupczyk/ngx-super-croppie/blob/develop/projects/ngx-super-croppie/package.json

**Comment by lukaskupczyk** [comment]
Thanks for bringing this issue to my attention. I have just released version 15.0.1, including a bug fix for this problem. Please let me know if this resolves the issue for you.

Best,
Lukas
